test/helper/routeHelper.ts

```diff
@@ -0,0 +1,25 @@
+import request from 'supertest'
+import express from 'express'
+
+export async function get(app: express.Express, endpoint: string) {
```

Comment:
why this is async? it returns a promise so no need .then/.catch and does not perform async within a function, can we just return?

We can return promise now?
```ts
export get = (...) => request(...
```

Reply:
updated to tidy them up